pioneers-testnet/pioneers-exercise-2.md

```diff
@@ -78,24 +82,20 @@ If you have any questions or encounter any problems, please feel free to use the
 
    The settings that are used here indicate that the transaction should be processed before slot 500,000 and  that it will cost no more than 1 Ada to submit (a safe value for a simple transaction on the Testnet).  You must pay this (small) fee every time you successfully process a transaction on the Cardano Blockchain.  It will be distributed as part of the pool rewards.  The source of this fee is encoded in the transaction as a UTxO.  We are now ready to sign the transaction and submit it to the chain.
 
-4. Generate a signing key, txsign.
```

Comment:
This was confusing to me, as I had it created in Ex 1, but if I misunderstood I can bring that section back